ai-player.js

```diff
@@ -249,6 +253,11 @@ function createAiPlayer(game, options) {
             return true;
         }
 
+        // Challenge if somebody claimed this role and lost
+        if (calledBluffs[state.state.playerIdx] && calledBluffs[state.state.playerIdx][claimedRole]) {
```

Comment:
I think this is not quite right: `state.state.playerIdx` is the player who took the initial action. If you are in ACTION_RESPONSE state, you need to check whether the *blocking player* (i.e., `state.state.target`) has been called on `claimedRole`.